platooning_tactical_plugin/include/platooning_tactical_plugin/log/log.h

```diff
@@ -0,0 +1,61 @@
+#include <sstream>
```

Comment:
I agree. It would be helpful to move most of the logic to a utils location, as they are common between most of the tactical plugins. However, I suggest we do this switch once the enhancements on ILC is complete. As for the platooning tactical, since the ILC logic is functional, I used that in this plugin so we can better troubleshoot the strategic and control platooning plugins.